serverless.yml

```diff
@@ -44,6 +44,8 @@ functions:
       OTP_SECRET: ${env:OTP_SECRET}
       OTP_EXPIRY: ${env:OTP_EXPIRY}
       FRONTEND_SENTRY_DSN: ${env:FRONTEND_SENTRY_DSN}
+      BACKEND_SENTRY_DSN: ${env:BACKEND_SENTRY_DSN}
+      SENTRY_ENVIRONMENT: ${env:SENTRY_ENVIRONMENT}
```

Comment:
I think instead of doing this, consider recycling `NODE_ENV` as defined above by removing this line like thus, then injecting ...
```suggestion

```